CHANGELOG.md

```diff
@@ -16,7 +16,12 @@ Release Versions:
 
 - feat: improve logging in parameter translators (#65)
 - fix(component-interface): make python subscriptions type safe (#71)
-- build: change base workspace image version
+- build: change base workspace image version (#83)
+- build: aica-package.toml for package-builder:v1 (#91)
+
+### Dependencies
+
+- control-libraries: 7.4.0 -> 8.1.0 (#91)
```

Comment:
You want to break main now? I thought we'd wait with this

Reply:
I didn't plan to release it imminently but I assumed we were gearing up for the breaking release soon. Although it's true that you are staging your other changes on a `breaking` branch currently. In any case we can wait to merge, I just wanted to get it ready here.

Reply:
Gearing up yes, but just wanted to check if we merge this right away. I see two options:
1. Finish the modulo controllers feature, release that as 4.2 and then start breaking things
2. Tag the current state of main as 4.1.1 and then start breaking things